packages/control/chargelog/chargelog.py

```diff
@@ -113,6 +115,14 @@ def save_and_reset_data(chargepoint, charging_ev, immediately: bool = True):
         log.exception("Fehler im Ladelog-Modul")
 
 
+def get_value_or_default(func):
+    try:
+        return func()
+    except Exception:
+        log.exception(f"Error getting value für chargelog: {func}. Setting to default.")
+        return "-"
```

Comment:
```suggestion
def get_value_or_default(func, default_value = "-"):
    try:
        return func()
    except Exception:
        log.exception(f"Error getting value for chargelog: {func}. Setting to default.")
        return default_value
```
Wenn man den Standardwert übergeben kann, wird die Methode wesentlich vielseitiger nutzbar.
In dem Zuge sollte überdacht werden, ob `-` ein sinnvoller Standardwert für eine Variable ist, die eigentlich eine Zahl ist. Für "undefiniert" oder "ungültig" würde ich eher `None` verwenden.

Reply:
umgesetzt